apps/crc_interactive.py

```diff
@@ -77,6 +77,13 @@ def __init__(self) -> None:
         additional_args.add_argument('-f', '--feature', help='specify a feature, e.g. `ti` for GPUs')
         additional_args.add_argument('-o', '--openmp', action='store_true', help='run using OpenMP style submission')
 
+    def parse_time(time_str: str) -> time:
```

Comment:
Neither `time` nor `ArgumentTypeError` has been imported. Add an import statement for each of them:

```python
from argparse import Namespace, ArgumentTypeError
from datetime import time
```

Also the `datetime.datetime` class is no longer needed in the imports.